modules/nw-metallb-loglevel.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-troubleshoot-support.adoc
+
+[id="nw-metallb-setting-metalb-logging-levels_{context}"]
+= Setting the MetalLB logging levels
+
+MetalLB uses FRRouting (FRR) in a container that by default generates a lot of logging. You can control the verbosity of the logs generated by setting the `logLevel` as illustrated in this example. 
+
+Gain a deeper insight into MetalLB by setting the `logLevel` to `debug` as follows:
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a file, such as `setdebugloglevel.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: MetalLB
+metadata:
+  name: metallb
+  namespace: metallb-system
+spec:
+  logLevel: debug
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+----
+
+. Apply the configuration:
++
+[source,terminal]
+----
+$ oc replace -f setdebugloglevel.yaml
+----
++
+[NOTE]
+====
+Use `oc replace` as the understanding is the metallb CR is already created and here you are changing the log level.
+====
+
+. Display the names of the `speaker` pods:
++
+[source,terminal]
+----
+$ oc get -n metallb-system pods -l app.kubernetes.io/component=speaker
+----
++
+.Example output
+[source,text]
+----
+NAME                    READY   STATUS    RESTARTS   AGE
+speaker-2m9pm           4/4     Running   0          9m19s
+speaker-7m4qw           3/4     Running   0          19s
+speaker-szlmx           4/4     Running   0          9m19s
+----
++
+[NOTE]
+====
+Speaker pods are recreated to ensure the updated logging level is applied.
```

Comment:
Controller as well. The logging level is modified for messaged logged in controller pod.